marimo/_config/config.py

```diff
@@ -89,6 +89,18 @@ class DisplayConfig(TypedDict, total=False):
     theme: Literal["light", "dark"]
 
 
+class BrowserConfig(TypedDict, total=False):
+    """Settings related to web browsers.
+
+    **Keys.**
+
+    - `browser`: `"default"` or a browser registered with Python's
+        webbrowser module (e.g., `"firefox"` or `"google-chrome"`)
+    """
+
+    browser: Literal["default"] | str
```

Comment:
List of supported values is here: https://docs.python.org/3/library/webbrowser.html#webbrowser.register. Figured it was better to just type as `str` rather than have to maintain exhaustive `Literal` type.